python/hail/dataset.py

```diff
@@ -1097,16 +1120,19 @@ def concordance(self, right):
                 VariantDataset(self.hc, result._2()))
 
     def count(self, genotypes=False):
-        """Return number of samples, variants and genotypes.
+        """Gets number of samples, variants and genotypes in this vds as a dictionary with the fields ``"nSamples"``, ``"nVariants"``, and ``"nGenotypes"``.
 
         :param bool genotypes: If True, include number of called
-            genotypes and genotype call rate.
+            genotypes and genotype call rate as fields ``"nCalled"`` and ``"callRate"``.
+
+        :return: Returns a python dictionary with keys as described above.
```

Comment:
Just say: "Returns a Python dictionary of counts".  As described above is implicit.